src/User.ts

```diff
@@ -148,7 +148,7 @@ export class User {
    * new user who wants to get onboarded, respectively has no ETH or trustline.
    * @param username Name of new user who wants to get onboarded.
    * @param serializedKeystore Serialized [eth-lightwallet](https://github.com/ConsenSys/eth-lightwallet)
```

Comment:
why did you not change this here? this should also be serializedWallet now. It is handed over to load

Reply:
Also the eth-lightwallet comment is totally outdated, please remove that.

Reply:
Because I struggled to understand what this function was used for, but afaict it is only useful for an ethers wallet and not for an identity wallet. The serializedKeystore will work for an ethers wallet so I did not want to change the function because the way it is used in the mobile app would need to change accordingly.
I can do it if you wish.